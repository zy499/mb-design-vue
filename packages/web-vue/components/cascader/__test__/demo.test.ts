import demoTest from '../../../scripts/demo-test';

demoTest('cascader');
