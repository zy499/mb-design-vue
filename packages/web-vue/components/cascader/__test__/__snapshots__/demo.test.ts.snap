// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<cascader> demo: render [basic] correctly 1`] = `
"<div class=\\"mb-space mb-space-vertical\\">
  <div class=\\"mb-space-item\\" style=\\"margin-bottom: 24px;\\"><span class=\\"mb-select-view-single mb-select-view mb-select-view-size-medium\\" style=\\"width: 320px;\\"><!----><input class=\\"mb-select-view-input\\" readonly=\\"\\" placeholder=\\"Please select ...\\"><span class=\\"mb-select-view-value mb-select-view-value-hidden\\"><!----></span><span class=\\"mb-select-view-suffix\\"><!----><span class=\\"mb-select-view-icon\\"><svg viewBox=\\"0 0 48 48\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\" stroke=\\"currentColor\\" class=\\"mb-icon mb-icon-down mb-select-view-arrow-icon\\" stroke-width=\\"4\\" stroke-linecap=\\"butt\\" stroke-linejoin=\\"miter\\"><path d=\\"M39.6 17.443 24.043 33 8.487 17.443\\"></path></svg></span>
    <!----></span></span>
    <!---->
  </div>
  <div class=\\"mb-space-item\\"><span class=\\"mb-select-view-single mb-select-view mb-select-view-size-medium\\" style=\\"width: 320px;\\"><!----><input class=\\"mb-select-view-input mb-select-view-input-hidden\\" readonly=\\"\\" placeholder=\\"Please select ...\\"><span class=\\"mb-select-view-value\\">Beijing / ChaoYang / Datunli</span><span class=\\"mb-select-view-suffix\\"><!----><span class=\\"mb-select-view-icon\\"><svg viewBox=\\"0 0 48 48\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\" stroke=\\"currentColor\\" class=\\"mb-icon mb-icon-down mb-select-view-arrow-icon\\" stroke-width=\\"4\\" stroke-linecap=\\"butt\\" stroke-linejoin=\\"miter\\"><path d=\\"M39.6 17.443 24.043 33 8.487 17.443\\"></path></svg></span>
    <!----></span></span>
    <!---->
  </div>
</div>"
`;

exports[`<cascader> demo: render [check-strictly] correctly 1`] = `
"<div class=\\"mb-space mb-space-vertical\\">
  <div class=\\"mb-space-item\\" style=\\"margin-bottom: 24px;\\"><span class=\\"mb-select-view-single mb-select-view mb-select-view-size-medium\\" style=\\"width: 320px;\\"><!----><input class=\\"mb-select-view-input mb-select-view-input-hidden\\" readonly=\\"\\" placeholder=\\"Please select ...\\"><span class=\\"mb-select-view-value\\">Beijing</span><span class=\\"mb-select-view-suffix\\"><!----><span class=\\"mb-select-view-icon\\"><svg viewBox=\\"0 0 48 48\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\" stroke=\\"currentColor\\" class=\\"mb-icon mb-icon-down mb-select-view-arrow-icon\\" stroke-width=\\"4\\" stroke-linecap=\\"butt\\" stroke-linejoin=\\"miter\\"><path d=\\"M39.6 17.443 24.043 33 8.487 17.443\\"></path></svg></span>
    <!----></span></span>
    <!---->
  </div>
  <div class=\\"mb-space-item\\"><span class=\\"mb-select-view mb-select-view-size-medium mb-select-view-has-tag mb-select-view-has-suffix mb-select-view-multiple\\" style=\\"width: 320px;\\"><span class=\\"mb-select-view-mirror\\"></span>
    <!---->
    <transition-group-stub><span class=\\"mb-tag mb-tag-size-medium mb-tag-checked mb-select-view-tag\\"><!--v-if-->Beijing<span class=\\"mb-icon-hover mb-tag-icon-hover mb-tag-close-btn\\" role=\\"button\\" aria-label=\\"Close\\"><svg viewBox=\\"0 0 48 48\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\" stroke=\\"currentColor\\" class=\\"mb-icon mb-icon-close\\" stroke-width=\\"4\\" stroke-linecap=\\"butt\\" stroke-linejoin=\\"miter\\"><path d=\\"M9.857 9.858 24 24m0 0 14.142 14.142M24 24 38.142 9.858M24 24 9.857 38.142\\"></path></svg></span>
      <!--v-if--></span><input class=\\"mb-select-view-input\\" style=\\"width: 12px;\\">
    </transition-group-stub>
    <!----><span class=\\"mb-select-view-suffix\\"><!----><span class=\\"mb-select-view-icon\\"><svg viewBox=\\"0 0 48 48\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\" stroke=\\"currentColor\\" class=\\"mb-icon mb-icon-expand\\" style=\\"transform: rotate(-45deg);\\" stroke-width=\\"4\\" stroke-linecap=\\"butt\\" stroke-linejoin=\\"miter\\"><path d=\\"M7 26v14c0 .552.444 1 .996 1H22m19-19V8c0-.552-.444-1-.996-1H26\\"></path></svg></span>
    <!---->
    <!----></span></span>
    <!---->
  </div>
</div>"
`;

exports[`<cascader> demo: render [clear] correctly 1`] = `
"<span class=\\"mb-select-view-single mb-select-view mb-select-view-size-medium\\" style=\\"width: 320px;\\"><!----><input class=\\"mb-select-view-input mb-select-view-input-hidden\\" readonly=\\"\\" placeholder=\\"Please select ...\\"><span class=\\"mb-select-view-value\\">Beijing / ChaoYang / Datunli</span><span class=\\"mb-select-view-suffix\\"><span class=\\"mb-icon-hover mb-select-view-clear-btn\\"><svg viewBox=\\"0 0 48 48\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\" stroke=\\"currentColor\\" class=\\"mb-icon mb-icon-close\\" stroke-width=\\"4\\" stroke-linecap=\\"butt\\" stroke-linejoin=\\"miter\\"><path d=\\"M9.857 9.858 24 24m0 0 14.142 14.142M24 24 38.142 9.858M24 24 9.857 38.142\\"></path></svg></span><span class=\\"mb-select-view-icon\\"><svg viewBox=\\"0 0 48 48\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\" stroke=\\"currentColor\\" class=\\"mb-icon mb-icon-down mb-select-view-arrow-icon\\" stroke-width=\\"4\\" stroke-linecap=\\"butt\\" stroke-linejoin=\\"miter\\"><path d=\\"M39.6 17.443 24.043 33 8.487 17.443\\"></path></svg></span>
<!----></span></span>
<!---->"
`;

exports[`<cascader> demo: render [disabled] correctly 1`] = `
"<span class=\\"mb-select-view-single mb-select-view mb-select-view-size-medium\\" style=\\"width: 320px;\\"><!----><input class=\\"mb-select-view-input\\" readonly=\\"\\" placeholder=\\"Please select ...\\"><span class=\\"mb-select-view-value mb-select-view-value-hidden\\"><!----></span><span class=\\"mb-select-view-suffix\\"><!----><span class=\\"mb-select-view-icon\\"><svg viewBox=\\"0 0 48 48\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\" stroke=\\"currentColor\\" class=\\"mb-icon mb-icon-down mb-select-view-arrow-icon\\" stroke-width=\\"4\\" stroke-linecap=\\"butt\\" stroke-linejoin=\\"miter\\"><path d=\\"M39.6 17.443 24.043 33 8.487 17.443\\"></path></svg></span>
<!----></span></span>
<!---->"
`;

exports[`<cascader> demo: render [expand] correctly 1`] = `
"<span class=\\"mb-select-view-single mb-select-view mb-select-view-size-medium\\" style=\\"width: 320px;\\"><!----><input class=\\"mb-select-view-input\\" readonly=\\"\\" placeholder=\\"Please select ...\\"><span class=\\"mb-select-view-value mb-select-view-value-hidden\\"><!----></span><span class=\\"mb-select-view-suffix\\"><!----><span class=\\"mb-select-view-icon\\"><svg viewBox=\\"0 0 48 48\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\" stroke=\\"currentColor\\" class=\\"mb-icon mb-icon-down mb-select-view-arrow-icon\\" stroke-width=\\"4\\" stroke-linecap=\\"butt\\" stroke-linejoin=\\"miter\\"><path d=\\"M39.6 17.443 24.043 33 8.487 17.443\\"></path></svg></span>
<!----></span></span>
<!---->"
`;

exports[`<cascader> demo: render [fallback] correctly 1`] = `
"<div class=\\"mb-space mb-space-vertical\\">
  <div class=\\"mb-space-item\\" style=\\"margin-bottom: 24px;\\"><span class=\\"mb-select-view mb-select-view-size-medium mb-select-view-has-tag mb-select-view-has-suffix mb-select-view-multiple\\" style=\\"width: 320px;\\"><span class=\\"mb-select-view-mirror\\"></span>
    <!---->
    <transition-group-stub><span class=\\"mb-tag mb-tag-size-medium mb-tag-checked mb-select-view-tag\\"><!--v-if-->Beijing / ChaoYang / Datunli<span class=\\"mb-icon-hover mb-tag-icon-hover mb-tag-close-btn\\" role=\\"button\\" aria-label=\\"Close\\"><svg viewBox=\\"0 0 48 48\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\" stroke=\\"currentColor\\" class=\\"mb-icon mb-icon-close\\" stroke-width=\\"4\\" stroke-linecap=\\"butt\\" stroke-linejoin=\\"miter\\"><path d=\\"M9.857 9.858 24 24m0 0 14.142 14.142M24 24 38.142 9.858M24 24 9.857 38.142\\"></path></svg></span>
      <!--v-if--></span><span class=\\"mb-tag mb-tag-size-medium mb-tag-checked mb-select-view-tag\\"><!--v-if-->wuhou<span class=\\"mb-icon-hover mb-tag-icon-hover mb-tag-close-btn\\" role=\\"button\\" aria-label=\\"Close\\"><svg viewBox=\\"0 0 48 48\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\" stroke=\\"currentColor\\" class=\\"mb-icon mb-icon-close\\" stroke-width=\\"4\\" stroke-linecap=\\"butt\\" stroke-linejoin=\\"miter\\"><path d=\\"M9.857 9.858 24 24m0 0 14.142 14.142M24 24 38.142 9.858M24 24 9.857 38.142\\"></path></svg></span>
      <!--v-if--></span><input class=\\"mb-select-view-input\\" style=\\"width: 12px;\\">
    </transition-group-stub>
    <!----><span class=\\"mb-select-view-suffix\\"><!----><span class=\\"mb-select-view-icon\\"><svg viewBox=\\"0 0 48 48\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\" stroke=\\"currentColor\\" class=\\"mb-icon mb-icon-expand\\" style=\\"transform: rotate(-45deg);\\" stroke-width=\\"4\\" stroke-linecap=\\"butt\\" stroke-linejoin=\\"miter\\"><path d=\\"M7 26v14c0 .552.444 1 .996 1H22m19-19V8c0-.552-.444-1-.996-1H26\\"></path></svg></span>
    <!---->
    <!----></span></span>
    <!---->
  </div>
  <div class=\\"mb-space-item\\"><span class=\\"mb-select-view mb-select-view-size-medium mb-select-view-has-tag mb-select-view-has-suffix mb-select-view-multiple\\" style=\\"width: 320px;\\"><span class=\\"mb-select-view-mirror\\"></span>
    <!---->
    <transition-group-stub><span class=\\"mb-tag mb-tag-size-medium mb-tag-checked mb-select-view-tag\\"><!--v-if-->Beijing / ChaoYang / Datunli<span class=\\"mb-icon-hover mb-tag-icon-hover mb-tag-close-btn\\" role=\\"button\\" aria-label=\\"Close\\"><svg viewBox=\\"0 0 48 48\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\" stroke=\\"currentColor\\" class=\\"mb-icon mb-icon-close\\" stroke-width=\\"4\\" stroke-linecap=\\"butt\\" stroke-linejoin=\\"miter\\"><path d=\\"M9.857 9.858 24 24m0 0 14.142 14.142M24 24 38.142 9.858M24 24 9.857 38.142\\"></path></svg></span>
      <!--v-if--></span><span class=\\"mb-tag mb-tag-size-medium mb-tag-checked mb-select-view-tag\\"><!--v-if-->SICHUAN-CHENGDU-WUHOU<span class=\\"mb-icon-hover mb-tag-icon-hover mb-tag-close-btn\\" role=\\"button\\" aria-label=\\"Close\\"><svg viewBox=\\"0 0 48 48\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\" stroke=\\"currentColor\\" class=\\"mb-icon mb-icon-close\\" stroke-width=\\"4\\" stroke-linecap=\\"butt\\" stroke-linejoin=\\"miter\\"><path d=\\"M9.857 9.858 24 24m0 0 14.142 14.142M24 24 38.142 9.858M24 24 9.857 38.142\\"></path></svg></span>
      <!--v-if--></span><input class=\\"mb-select-view-input\\" style=\\"width: 12px;\\">
    </transition-group-stub>
    <!----><span class=\\"mb-select-view-suffix\\"><!----><span class=\\"mb-select-view-icon\\"><svg viewBox=\\"0 0 48 48\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\" stroke=\\"currentColor\\" class=\\"mb-icon mb-icon-expand\\" style=\\"transform: rotate(-45deg);\\" stroke-width=\\"4\\" stroke-linecap=\\"butt\\" stroke-linejoin=\\"miter\\"><path d=\\"M7 26v14c0 .552.444 1 .996 1H22m19-19V8c0-.552-.444-1-.996-1H26\\"></path></svg></span>
    <!---->
    <!----></span></span>
    <!---->
  </div>
</div>"
`;

exports[`<cascader> demo: render [field-names] correctly 1`] = `
"<span class=\\"mb-select-view-single mb-select-view mb-select-view-size-medium\\" style=\\"width: 320px;\\"><!----><input class=\\"mb-select-view-input\\" readonly=\\"\\" placeholder=\\"Please select ...\\"><span class=\\"mb-select-view-value mb-select-view-value-hidden\\"><!----></span><span class=\\"mb-select-view-suffix\\"><!----><span class=\\"mb-select-view-icon\\"><svg viewBox=\\"0 0 48 48\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\" stroke=\\"currentColor\\" class=\\"mb-icon mb-icon-down mb-select-view-arrow-icon\\" stroke-width=\\"4\\" stroke-linecap=\\"butt\\" stroke-linejoin=\\"miter\\"><path d=\\"M39.6 17.443 24.043 33 8.487 17.443\\"></path></svg></span>
<!----></span></span>
<!---->"
`;

exports[`<cascader> demo: render [format] correctly 1`] = `
"<span class=\\"mb-select-view-single mb-select-view mb-select-view-size-medium\\" style=\\"width: 320px;\\"><!----><input class=\\"mb-select-view-input mb-select-view-input-hidden\\" readonly=\\"\\" placeholder=\\"Please select ...\\"><span class=\\"mb-select-view-value\\">Beijing-ChaoYang-Datunli</span><span class=\\"mb-select-view-suffix\\"><!----><span class=\\"mb-select-view-icon\\"><svg viewBox=\\"0 0 48 48\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\" stroke=\\"currentColor\\" class=\\"mb-icon mb-icon-down mb-select-view-arrow-icon\\" stroke-width=\\"4\\" stroke-linecap=\\"butt\\" stroke-linejoin=\\"miter\\"><path d=\\"M39.6 17.443 24.043 33 8.487 17.443\\"></path></svg></span>
<!----></span></span>
<!---->"
`;

exports[`<cascader> demo: render [lazy-load] correctly 1`] = `
"<div class=\\"mb-space mb-space-horizontal mb-space-align-center\\">
  <div class=\\"mb-space-item\\"><span class=\\"mb-select-view-single mb-select-view mb-select-view-size-medium\\" style=\\"width: 320px;\\"><!----><input class=\\"mb-select-view-input\\" readonly=\\"\\" placeholder=\\"Please select ...\\"><span class=\\"mb-select-view-value mb-select-view-value-hidden\\"><!----></span><span class=\\"mb-select-view-suffix\\"><!----><span class=\\"mb-select-view-icon\\"><svg viewBox=\\"0 0 48 48\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\" stroke=\\"currentColor\\" class=\\"mb-icon mb-icon-down mb-select-view-arrow-icon\\" stroke-width=\\"4\\" stroke-linecap=\\"butt\\" stroke-linejoin=\\"miter\\"><path d=\\"M39.6 17.443 24.043 33 8.487 17.443\\"></path></svg></span>
    <!----></span></span>
    <!---->
  </div>
</div>"
`;

exports[`<cascader> demo: render [loading] correctly 1`] = `
"<span class=\\"mb-select-view-single mb-select-view mb-select-view-size-medium\\" style=\\"width: 320px;\\"><!----><input class=\\"mb-select-view-input\\" readonly=\\"\\" placeholder=\\"Please select ...\\"><span class=\\"mb-select-view-value mb-select-view-value-hidden\\"><!----></span><span class=\\"mb-select-view-suffix\\"><!----><span class=\\"mb-select-view-icon\\"><svg viewBox=\\"0 0 48 48\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\" stroke=\\"currentColor\\" class=\\"mb-icon mb-icon-loading\\" stroke-width=\\"4\\" stroke-linecap=\\"butt\\" stroke-linejoin=\\"miter\\"><path d=\\"M42 24c0 9.941-8.059 18-18 18S6 33.941 6 24 14.059 6 24 6\\"></path></svg></span>
<!----></span></span>
<!---->"
`;

exports[`<cascader> demo: render [multiple] correctly 1`] = `
"<span class=\\"mb-select-view mb-select-view-size-medium mb-select-view-has-tag mb-select-view-has-suffix mb-select-view-multiple\\" style=\\"width: 320px;\\"><span class=\\"mb-select-view-mirror\\"></span>
<!---->
<transition-group-stub><span class=\\"mb-tag mb-tag-size-medium mb-tag-checked mb-select-view-tag\\"><!--v-if-->Beijing / ChaoYang / Datunli<span class=\\"mb-icon-hover mb-tag-icon-hover mb-tag-close-btn\\" role=\\"button\\" aria-label=\\"Close\\"><svg viewBox=\\"0 0 48 48\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\" stroke=\\"currentColor\\" class=\\"mb-icon mb-icon-close\\" stroke-width=\\"4\\" stroke-linecap=\\"butt\\" stroke-linejoin=\\"miter\\"><path d=\\"M9.857 9.858 24 24m0 0 14.142 14.142M24 24 38.142 9.858M24 24 9.857 38.142\\"></path></svg></span>
  <!--v-if--></span><input class=\\"mb-select-view-input\\" style=\\"width: 12px;\\">
</transition-group-stub>
<!----><span class=\\"mb-select-view-suffix\\"><!----><span class=\\"mb-select-view-icon\\"><svg viewBox=\\"0 0 48 48\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\" stroke=\\"currentColor\\" class=\\"mb-icon mb-icon-expand\\" style=\\"transform: rotate(-45deg);\\" stroke-width=\\"4\\" stroke-linecap=\\"butt\\" stroke-linejoin=\\"miter\\"><path d=\\"M7 26v14c0 .552.444 1 .996 1H22m19-19V8c0-.552-.444-1-.996-1H26\\"></path></svg></span>
<!---->
<!----></span></span>
<!---->"
`;

exports[`<cascader> demo: render [panel] correctly 1`] = `
"<transition-group-stub>
  <div class=\\"mb-cascader-panel-column\\" style=\\"z-index: 3;\\">
    <ul role=\\"menu\\" class=\\"mb-cascader-list\\">
      <li tabindex=\\"0\\" role=\\"menuitem\\" aria-disabled=\\"false\\" aria-haspopup=\\"true\\" aria-expanded=\\"false\\" title=\\"Beijing\\" class=\\"mb-cascader-option\\">
        <!---->
        <!---->
        <div class=\\"mb-cascader-option-label\\">Beijing<svg viewBox=\\"0 0 48 48\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\" stroke=\\"currentColor\\" class=\\"mb-icon mb-icon-right\\" stroke-width=\\"4\\" stroke-linecap=\\"butt\\" stroke-linejoin=\\"miter\\">
            <path d=\\"m16 39.513 15.556-15.557L16 8.4\\"></path>
          </svg></div>
      </li>
      <li tabindex=\\"0\\" role=\\"menuitem\\" aria-disabled=\\"false\\" aria-haspopup=\\"true\\" aria-expanded=\\"false\\" title=\\"Shanghai\\" class=\\"mb-cascader-option\\">
        <!---->
        <!---->
        <div class=\\"mb-cascader-option-label\\">Shanghai<svg viewBox=\\"0 0 48 48\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\" stroke=\\"currentColor\\" class=\\"mb-icon mb-icon-right\\" stroke-width=\\"4\\" stroke-linecap=\\"butt\\" stroke-linejoin=\\"miter\\">
            <path d=\\"m16 39.513 15.556-15.557L16 8.4\\"></path>
          </svg></div>
      </li>
    </ul>
  </div>
</transition-group-stub>"
`;

exports[`<cascader> demo: render [path] correctly 1`] = `
"<div class=\\"mb-space mb-space-vertical\\">
  <div class=\\"mb-space-item\\" style=\\"margin-bottom: 24px;\\"><span class=\\"mb-select-view-single mb-select-view mb-select-view-size-medium\\" style=\\"width: 320px;\\"><!----><input class=\\"mb-select-view-input\\" readonly=\\"\\" placeholder=\\"Please select ...\\"><span class=\\"mb-select-view-value mb-select-view-value-hidden\\"><!----></span><span class=\\"mb-select-view-suffix\\"><!----><span class=\\"mb-select-view-icon\\"><svg viewBox=\\"0 0 48 48\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\" stroke=\\"currentColor\\" class=\\"mb-icon mb-icon-down mb-select-view-arrow-icon\\" stroke-width=\\"4\\" stroke-linecap=\\"butt\\" stroke-linejoin=\\"miter\\"><path d=\\"M39.6 17.443 24.043 33 8.487 17.443\\"></path></svg></span>
    <!----></span></span>
    <!---->
  </div>
  <div class=\\"mb-space-item\\"><span class=\\"mb-select-view-single mb-select-view mb-select-view-size-medium\\" style=\\"width: 320px;\\"><!----><input class=\\"mb-select-view-input mb-select-view-input-hidden\\" readonly=\\"\\" placeholder=\\"Please select ...\\"><span class=\\"mb-select-view-value\\">Beijing / ChaoYang / Datunli</span><span class=\\"mb-select-view-suffix\\"><!----><span class=\\"mb-select-view-icon\\"><svg viewBox=\\"0 0 48 48\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\" stroke=\\"currentColor\\" class=\\"mb-icon mb-icon-down mb-select-view-arrow-icon\\" stroke-width=\\"4\\" stroke-linecap=\\"butt\\" stroke-linejoin=\\"miter\\"><path d=\\"M39.6 17.443 24.043 33 8.487 17.443\\"></path></svg></span>
    <!----></span></span>
    <!---->
  </div>
</div>"
`;

exports[`<cascader> demo: render [search] correctly 1`] = `
"<span class=\\"mb-select-view-single mb-select-view mb-select-view-size-medium mb-select-view-search\\" style=\\"width: 320px;\\"><!----><input class=\\"mb-select-view-input\\" placeholder=\\"Please select ...\\"><span class=\\"mb-select-view-value mb-select-view-value-hidden\\"><!----></span><span class=\\"mb-select-view-suffix\\"><!----><span class=\\"mb-select-view-icon\\"><svg viewBox=\\"0 0 48 48\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\" stroke=\\"currentColor\\" class=\\"mb-icon mb-icon-expand\\" style=\\"transform: rotate(-45deg);\\" stroke-width=\\"4\\" stroke-linecap=\\"butt\\" stroke-linejoin=\\"miter\\"><path d=\\"M7 26v14c0 .552.444 1 .996 1H22m19-19V8c0-.552-.444-1-.996-1H26\\"></path></svg></span>
<!----></span></span>
<!---->"
`;
